.github/workflows/azure-devops-pipelines-cg.yml

```diff
@@ -0,0 +1,49 @@
+# This pipeline is intended to use inside Azure DevOps for Component Governance,
+# which is may not useful for users outside Microsoft.
+
+variables:
+- name: BuildParameters.solution
+  value: '**\*.sln'
```

Comment:
You reference `BuildParameters.solution` below in the last step, so why are these defined as variables?

Reply:
This yaml is exported from our classic pipeline. Anyway, I can change it.